src/kakarot/accounts/eoa/library.cairo

```diff
@@ -70,7 +72,7 @@ namespace ExternallyOwnedAccount {
         return (evm_address=address);
     }
 
-    // @notice Check if tx is signed and valid for each call.
+    // @notice Check if tx is signed and valid for each call and validate the transaction.
```

Comment:
what does "validate" a tx means of not checking that the signature is valid? You could however give more details about how this is done:
- the whole EVM signed tx is given in the calldata
- it is rlp decoded, then message hash is computed and signature is checked against this hash (see `EthTransaction.validate` for more details)